### ISSUE TITLE
pasting formatted text should not be possible when in context editable mode - but only when an element is active.

### ISSUE THREAD

**Issue by atulbhatt-system32** [rnbwdev/rnbw#590]

**Comment by atulbhatt-system32** [comment]
@edenvidal in this case the pasting of content using `cmd+v` should be disabled. Is it so?

**Comment by edenvidal** [comment]
true, when already editing-content, the pasted text should join inside in a non-formatted way.

**Comment by atulbhatt-system32** [comment]
@edenvidal It may be a trivial question but to be clear.

`cmd + v ` won't do anything inside content editable mode.

`cmd + shift + v` will only work if the user wants to copy.

**Comment by edenvidal** [comment]
no @atulbhatt-system32, it will paste the text, but unformatted.

**Comment by atulbhatt-system32** [comment]
@edenvidal got it. Last thing to clear about unformatted meaning.

If you want that when you type `<b>hello world</b>` inside a content editable field, it should show `<b>hello world</b>` only and not `hello world` in `bold` it is somewhat tricky.

Because whatever we type inside content editable becomes part of the HTML file code. And the parser parses it. To have it behave like a text and not the HTML element we would have to manually replace all the `< and >` in copied text with `&gt; and &lt;` which is possible but it is what we would have to do. However, it won't be something the user copied.

**Comment by edenvidal** [comment]
@atulbhatt-system32 right, when you type down code inside content editable it should alternate its characters and not as actual code.

**Comment by atulbhatt-system32** [comment]
@edenvidal this is up for review
